django_app/frontend/src/js/main.js

```diff
@@ -1,3 +1,18 @@
 import "./trusted-types.js";
 import "./libs/govuk-frontend.min.js";
 import "../../node_modules/i.ai-design-system/dist/iai-design-system.js";
+
+// Because this doesn't appear to be working in Redbox: https://design-system.service.gov.uk/components/button/#stop-users-from-accidentally-sending-information-more-than-once
+(() => {
+  let buttons = document.querySelectorAll(`[data-prevent-double-click="true"]`);
+  buttons.forEach((button) => {
+    button.addEventListener("click", () => {
+      window.setTimeout(() => {
```

Comment:
Is the first timeout needed?

Reply:
Seemingly so. I first tried it without and it prevented anything from happening! Leaving it until further down the queue seems to solve that one.